changelogs/fragments/255-3.0.0-deprecations.yml

```
@@ -0,0 +1,4 @@
+---
+removed_features:
+  - "The deprecated ``community.network.sros`` plugin has been removed. Use ``nokia.sros.md`` instead."
```

```suggestion
  - "The deprecated ``community.network.sros`` netconf plugin has been removed. Use ``nokia.sros.md`` instead (https://github.com/ansible-collections/community.network/pull/255)."
```
For plugins it is important to mention the plugin type.